### PR TITLE
dir: Prefer unmaintained related extensions

### PR DESCRIPTION
**Pull request by jprvita**
If an extension is already installed as an unmainted extension, prefer
it instead of downloading from the remote.

### CODE REVIEW

File: common/flatpak-dir.c [hunk 1]
```diff
@@ -9876,6 +9877,19 @@ add_related (FlatpakDir *self,
...
 
+  /* Don't download if there is an unmaintained extension already installed */
+  unmaintained_path =
+    flatpak_find_unmaintained_extension_dir_if_exists (ref_parts[1],
+                                                       ref_parts[2],
+                                                       ref_parts[3], NULL);
+  if (unmaintained_path != NULL && deploy_data == NULL)
+    {
+      g_debug ("Skipping related extension ‘%s’ because it is already "
+               "installed as an unmaintaned extension in ‘%s’.",
```
Comment (AsciiWolf) [inline common/flatpak-dir.c]: Typo: unmaintaned -> unmaintained

Comment (jprvita) [reply common/flatpak-dir.c]: Thanks, should be fixed now.

**Comment by alexlarsson** [comment]
bot, add author to whitelist

**Comment by alexlarsson** [comment]
@rh-atomic-bot r+

**Pull request closed without merging**